server/src/main/java/org/apache/iotdb/db/protocol/influxdb/meta/InfluxDBMetaManager.java

```diff
@@ -201,15 +198,15 @@ private void updateTagInfoRecords(TagInfoRecords tagInfoRecords) {
     List<InsertRowPlan> plans = tagInfoRecords.convertToInsertRowPlans();
     for (InsertRowPlan plan : plans) {
       try {
-        basicServiceProvider.executeNonQuery(plan);
+        serviceProvider.executeNonQuery(plan);
       } catch (QueryProcessException | StorageGroupNotSetException | StorageEngineException e) {
         throw new InfluxDBException(e.getMessage());
       }
     }
   }
 
   private static class MetaManagerHolder {
```

Comment:
change to `InfluxDBMetaManagerHolder`?

Reply:
Thanks for your review, the reply is a little slow because of the recent course examination. :D

It has been modified to InfluxDBMetaManagerHolder ~